build.gradle

```diff
@@ -1,6 +1,6 @@
 group = 'org.interledger'
 description = 'Interledger Protocol Core'
-version = '0.5.0-SNAPSHOT'
+version = '0.5.1-SNAPSHOT'
```

Comment:
Not sure when this PR will be fully-baked, but it's likely that #55 will be merged before this one, so something to keep in mind - you'll need to pull-in those changes to this build file.